h/presenters/group_json.py

```diff
@@ -28,6 +28,23 @@ def _expand(self, model, expand):
                 model["organization"] = OrganizationJSONPresenter(
                     self.organization_context
                 ).asdict()
+        if "scopes" in expand:
```

Comment:
I notice that we don't return any error if the user supplies an invalid value for an `expand` property. As a separate task, I wonder if we should consider changing that to make errors in the query more obvious?

Reply:
I opted to take an ignore-not-raise approach on malformed or invalid `expand` parameters. This feels appropriate for a `GET` endpoint to me, e.g. annotation-search has the same behavior. Note that this endpoint won't raise on a malformed `document_uri` either (it'll just ignore it/not match scoped groups). So there is consistency here...